dist/images/ovnkube.sh

```diff
@@ -2399,8 +2422,13 @@ ovn-node() {
     wait_for_event ovs_ready
   fi
 
-  echo "=============== ovn-node - (wait for ready_to_start_node)"
-  wait_for_event ready_to_start_node
+  if [[ ${ovnkube_node_mode} != "dpu-host" ]] && [[ ${ovn_enable_interconnect} != "true" ]]; then
```

Comment:
ok. too fast, I think this may be the reason for test failure, should this condition use `||` instead of `&&`?

Reply:
fair point, will test on my cluster and lets see if the CI passes.